Detectors/HMPID/reconstruction/src/HmpidDecoder.cxx

```diff
@@ -359,7 +359,7 @@ bool HmpidDecoder::decodeHmpidError(int ErrorField, char* outbuf)
 int HmpidDecoder::decodeHeader(uint32_t* streamPtrAdr, int* EquipIndex)
 {
   uint32_t* buffer = streamPtrAdr; // Sets the pointer to buffer
-  o2::header::RAWDataHeaderV6* hpt = (o2::header::RAWDataHeaderV6*)buffer;
+  o2::header::RAWDataHeaderV7* hpt = (o2::header::RAWDataHeaderV7*)buffer;
```

Comment:
Idem, please use `o2::header::RAWDataHeader` which for you should be the same for all versions >= 5.